mypy.ini

```diff
@@ -0,0 +1,7 @@
+# Global options:
+[mypy]
+python_version = 3.7
+warn_return_any = True
```

Comment:
I'm guessing warn means literally - warn but do not error? Just want to make sure that at most, all it does it complain. Also don't know if we want users to feel forced to add type annotations, which might increase the barrier to them contributing. Don't think it will be an issue, though.

Reply:
I think it only warns, and doesn't throw a shell error.

Reply:
Actually, it exits with code 1.

Reply:
Hmm. Maybe the way to go then is to say to users that they don’t need to personally manage type annotations, but if they do, they need to make sure they pass?

Reply:
And we can add annotations in their PRs where useful before mergimg

Reply:
Reading your comment below, I definitely like the idea of a skeleton function and skeleton suite of tests.